### DISCUSSION TITLE
明确责任机制，开发商、监管等需要承担无限责任。个人破产机制，允许个人断供破产。

### DISCUSSION THREAD

**Discussion by xiaoyao-jimmy** [WeNeedHome/SummaryOfLoanSuspension#103] [General]
明确责任机制，开发商、监管等需要承担无限责任。个人破产机制，允许个人断供破产。

**Comment by SystemCodes** [comment]
别老是幻想不可能的事情好吗？

本来就是故意把风险转嫁消费者的办法。

就好比政府外包各种零时工，出力最多，突然有人提出要给他们编制，别开玩笑了

**Reply by luzhanmin** [reply]
所以，所谓的共产主义，已经变了颜色吗~~

**Reply by lei946** [reply]
人心是会变的，尤其权力上层没有得到监督的时候，能捞好处为什么不去捞。然后就变成官官相护，腐肉烂入骨

**Reply by ruo-xi** [reply]
> 中国特色之后 就已经变色了并且渐行渐远了，没什么办法。只能希望习真的有一颗红色的心把，而不是做做样子。

**Reply by zhaokai5520** [reply]
政府事业单位的外包更惨

**Reply by KHW2099** [reply]
> > 中国特色之后 就已经变色了并且渐行渐远了，没什么办法。只能希望习真的有一颗红色的心把，而不是做做样子。

红心？做做样子都没看到，志大才疏，你的希望让人绝望！

**Comment by zhaokai5520** [comment]
你要知道制度是谁设计的？

**Comment by CodeJiang** [comment]
虽然我是个怂逼,没太争取过自己的权利，
但是还是要说一句,权利是争取来的.

**Reply by Jenghan** [reply]
有一例外，土匪的權利是用暴力搶來的  (不值得學習)

**Comment by diaohancai** [comment]
个人破产机制，责任有限，那就是硬着陆了，不可能的

**Comment by chenxxing** [comment]
现在都变了，社会主义变得资本化了，而资本主义却变得社会化了

**Reply by blisee** [reply]
有资本的社会主义就是耍流氓

**Comment by CosmoJulis** [comment]
地方政府卖地拿了大头，然后就拍屁股走人了，本着谁受益谁负责的道理，地方政府应该提供贷款接收烂尾项目。
如果有银行监管资金违规操作的证据，地方政府应该无偿接收烂尾项目。

**Comment by Jesse3692** [comment]
害  你们还能咋滴，不痛不痒的  分化一批，拉拢一批，打压一批  接着奏乐接着舞

**Comment by Jesse3692** [comment]
又加上要开会了，能闹起来？

**Comment by yalo-ddaa** [comment]
除非各地停贷的人雪崩式增长，一直到了再用“征信”会影响整体经济的地步，才有可能出台楼主所述的政策。

**Comment by zhaipro** [comment]
需要保险吗？
投保后，房产没有按期交付可获得赔偿金，便宜卖啦。

**Comment by xiaer93** [comment]
办报自由、集会自由、官员财产公开、领导人普选……

师夷长技以制夷。

**Comment by Novite5** [comment]
四川高等法院有判例：业主不用负责，银行应该找地产商。
